meta-qoriq/recipes-extended/vpp/files/0048-vpp-version-format-change.patch

```diff
@@ -0,0 +1,21 @@
+diff -ruN a/vpp/app/version.c b/vpp/app/version.c
+--- a/vpp/app/version.c	2022-11-11 12:17:52.665882031 +0530
++++ b/vpp/app/version.c	2022-11-11 12:29:15.631962472 +0530
+@@ -29,7 +29,7 @@
+ 
+ /** The image version string */
+ char *vpe_version_string =
+-  "vpp v" VPP_BUILD_VER
++  "vpp v21.01 " "("VPP_BUILD_VER")"
```

Comment:
- Can you remove the redundant quotes?
```
"vpp v21.01 " "("  /* wrong */
"vpp v21.01 ("  /* correct */
```
- Make sure the original `VPP_BUILD_VER` has a "v" prefix as well for consistency, not just the new string.
- I suggest writing `#define VPP_DISPLAY_VER 21.01` and using that define in both places instead.